addons/project/__manifest__.py

```diff
@@ -117,12 +117,10 @@
 
             # Remove Longpolling bus and packages needed this bus
             ('remove', 'bus/static/src/js/services/assets_watchdog_service.js'),
-            ('remove', 'mail/static/src/services/messaging/messaging.js'),
+            ('remove', 'mail/static/src/services/messaging_service.js'),
```

Comment:
wtf... this module removing stuff, instead of splitting the main bundle into relevant parts...